File: JAVA/Class.java
class Film {
    private String id;
    private String judul;
    private String genre;
    private String sutradara;
    private int durasi; 
    private int harga; 

    public Film() {
        this.id = "";
        this.judul = "";
        this.genre = "";
        this.sutradara = "";
        this.durasi = 0;
        this.harga = 0;
    }

    public Film(String id, String judul, String genre, String sutradara, int durasi, int harga) {
        this.id = id;
        this.judul = judul;
        this.genre = genre;
        this.sutradara = sutradara;
        this.durasi = durasi;
        this.harga = harga;
    }

    public void setId(String id) { this.id = id; }
    public String getId() { return this.id; }

    public void setJudul(String judul) { this.judul = judul; }
    public String getJudul() { return this.judul; }

    public void setGenre(String genre) { this.genre = genre; }
    public String getGenre() { return this.genre; }

    public void setSutradara(String sutradara) { this.sutradara = sutradara; }
    public String getSutradara() { return this.sutradara; }

    public void setDurasi(int durasi) { this.durasi = durasi; }
    public int getDurasi() { return this.durasi; }

    public void setHarga(int harga) { this.harga = harga; }
    public int getHarga() { return this.harga; }

    public void tampilkanFilm() {
        System.out.println("ID Film   : " + id);
        System.out.println("Judul     : " + judul);
        System.out.println("Genre     : " + genre);
        System.out.println("Sutradara : " + sutradara);
        System.out.println("Durasi    : " + durasi + " menit");
        System.out.println("Harga     : Rp " + harga);
        System.out.println("--------------------------------");
    }
}

File: PYTHON/Class.py
class Film:
    def __init__(self, id: str = "", judul: str = "", genre: str = "", sutradara: str = "", durasi: int = 0, harga: int = 0):
        self._id = str(id)
        self._judul = str(judul)
        self._genre = str(genre)
        self._sutradara = str(sutradara)
        self._durasi = int(durasi)
        self._harga = int(harga)

    def setId(self, id: str) -> None:
        self._id = id

    def getId(self) -> str:
        return self._id

    def setJudul(self, judul: str) -> None:
        self._judul = judul

    def getJudul(self) -> str:
        return self._judul

    def setGenre(self, genre: str) -> None:
        self._genre = genre

    def getGenre(self) -> str:
        return self._genre

    def setSutradara(self, sutradara: str) -> None:
        self._sutradara = sutradara

    def getSutradara(self) -> str:
        return self._sutradara

    def setDurasi(self, durasi: int) -> None:
        self._durasi = durasi

    def getDurasi(self) -> int:
        return self._durasi

    def setHarga(self, harga: int) -> None:
        self._harga = harga

    def getHarga(self) -> int:
        return self._harga

    def tampilkanFilm(self) -> None:
        print(f"ID Film   : {self._id}")
        print(f"Judul     : {self._judul}")
        print(f"Genre     : {self._genre}")
        print(f"Sutradara : {self._sutradara}")
        print(f"Durasi    : {self._durasi} menit")
        print(f"Harga     : Rp {self._harga}")
        print("--------------------------------")
